tests/test_chimera.cpp

```diff
@@ -0,0 +1,408 @@
+    0,                                          /* tp_as_mapping */
+    0,                                          /* tp_hash */
+    0,                                          /* tp_call */
+    0,                                          /* tp_str */
+    &PyChimera_getattro,                        /* tp_getattro */
+    &PyChimera_setattro,                        /* tp_setattro */
+    0,                                          /* tp_as_buffer */
+    Py_TPFLAGS_DEFAULT,                         /* tp_flags */
+    "Chimera objects",                          /* tp_doc */
+    0,                                          /* tp_traverse */
+    0,                                          /* tp_clear */
+    0,                                          /* tp_richcompare */
+    0,                                          /* tp_weaklistoffset */
+    0,                                          /* tp_iter */
+    0,                                          /* tp_iternext */
+    0,                                          /* tp_methods */
+    0,                                          /* tp_members */
+    0,                                          /* tp_getset */
+    0,                                          /* tp_base */
+    0,                                          /* tp_dict */
+    0,                                          /* tp_descr_get */
+    0,                                          /* tp_descr_set */
+    0,                                          /* tp_dictoffset */
+    0,                                          /* tp_init */
+    0,                                          /* tp_alloc */
+    0,                                          /* tp_new */
+    0,                                          /* tp_free */
+    0,                                          /* tp_is_gc */
+    0,                                          /* tp_bases */
+    0,                                          /* tp_mro */
+    0,                                          /* tp_cache */
+    0,                                          /* tp_subclasses */
+    0,                                          /* tp_weaklist */
+    0,                                          /* tp_del */
+    0,                                          /* tp_version_tag */
+#if PY_VERSION_HEX >= 0x030400a1
+    0,                                          /* tp_finalize */
+#endif
+#if PY_VERSION_HEX >= 0x030800b1
+    0,                                          /* tp_vectorcall */
+#endif
+#if PY_VERSION_HEX >= 0x030800b4 && PY_VERSION_HEX < 0x03090000
+    0,                                          /* tp_print */
+#endif
+#if defined(PYPY_VERSION)
+    0,                                          /* tp_pypy_flags */
+#endif
+};
+
+
+#if defined(PY38_WARNING_WORKAROUND_ENABLED)
+#undef PY38_WARNING_WORKAROUND_ENABLED
+#pragma GCC diagnostic pop
+#endif
+
+static std::unordered_map<Chimera*, void*>* mapping =
```

Comment:
The shorter
`static std::unordered_map<Chimera*, void*> mapping;`
(with `mapping->` below replaced by `mapping`) also works. Is there a strong reason for allocating `mapping` on the heap? (Its size is only 56 bytes in a 64-bit build.)

Similarly for the `shared` and `shared_const` variables below.

Reply:
Done.

Reply:
Done.